docs/body.md

```diff
@@ -1292,6 +1292,11 @@ important.
 - zWinDBSnapshotIgnore
     :   MS SQL Database Snapshots modeling disabled. Set to true to disable modeling and monitoring of MS SQL Database snapshots.
 
+- zWinServicesGroupedByClass
+    :   List of regular expressions for Windows services to model with generic Windows Service class. Specify class names to set generic Windows Service class names for matching Windows Services.
+        This property should be used when a target Windows device has a lot of Windows Services with rotating name extension - "<service_name>_0123456789bcdef". Each of these services creates separate Windows Service class which could cause ZODB lock.
```

Comment:
probably don't need to mention ZODB locking, but should mention that the regex is used in search() not match(), so not anchored to the start of the name, and that the regex supplied has no bearing on what is removed from the service name to arrive at the service class name, it simply removes the last underscore and everything following it.